packages/matrix-identity-server/src/config.json

```diff
@@ -41,6 +41,7 @@
   "smtp_sender": "",
   "smtp_server": "localhost",
   "smtp_port": 25,
+  "trust_x_forwarded_for": false,
```

Comment:
Did you see [my changes on server.mjs](https://github.com/linagora/ToM-server/commit/b81532f1) ?